include/sensor.h

```diff
@@ -112,6 +112,13 @@ enum sensor_channel {
 	SENSOR_CHAN_BLUE,
 	/** Altitude, in meters */
 	SENSOR_CHAN_ALTITUDE,
+
```

Comment:
Change your commit message and remove reference about pms7003

What you are doing here is adding generic channels for any Particulate Matter: on 1.0, 2.5 and 10 micrometer.